server/opts.go

```diff
@@ -2551,6 +2558,7 @@ func ConfigureOptions(fs *flag.FlagSet, args []string, printVersion, printHelp,
 	fs.StringVar(&opts.TLSCert, "tlscert", "", "Server certificate file.")
 	fs.StringVar(&opts.TLSKey, "tlskey", "", "Private key for server certificate.")
 	fs.StringVar(&opts.TLSCaCert, "tlscacert", "", "Client certificate CA for verification.")
+	fs.IntVar(&opts.MaxTracedMsgLen, "max_traced_msg_len", -1, "Maximum printable length for traced messages")
```

Comment:
This only sets via server binary. We have many people that embed the server in other Go apps so we need the default in that mode as well.

Reply:
What is the way for lib users to initialize an instance for options? i have no idea about to where to start?

Reply:
I have thought about this:
* Default value for `max_traces_msg_len` is 0. This should be also adapting to lib usage for unlimited message length
* Print traced message with customized length only when `MaxTracedMsgLen` is 0.

Reply:
@derekcollison PTAL, again.